api/src/db/migrations/v122_remove_welcome_and_welcomeupandrunning_sidecards.test.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  generateV121Business,
+  generateV121Preferences,
+  generateV121UserData,
+} from "./v121_add_nonprofit_formation_fields";
+import { migrate_v121_to_v122 } from "./v122_remove_welcome_and_welcomeupandrunning_sidecards";
+
+describe("migrate_v121_to_v122", () => {
+  it("removes the welcome and welcome-up-and-running from sidebarCards", () => {
+    //Setup
```

Comment:
[pebble] Comments are great to help guide you through the process of writing the test, but they should be removed before merging to main.

Reply:
removed